README.md

```diff
@@ -1,15 +1,60 @@
-# tket2
+# tket2: The Hardware Agnostic Quantum Compiler
 
 [![build_status][]](https://github.com/CQCL-DEV/tket2/actions)
 ![msrv][]
 [![codecov][]](https://codecov.io/gh/CQCL/tket2)
 
-Version 2 of the TKET compiler.
-
   [build_status]: https://github.com/CQCL-DEV/hugr/workflows/Continuous%20integration/badge.svg?branch=main
   [msrv]: https://img.shields.io/badge/rust-1.75.0%2B-blue.svg
   [codecov]: https://img.shields.io/codecov/c/gh/CQCL/tket2?logo=codecov
 
+TKET2 is an open source quantum compiler developed by Quantinuum. Central to
+TKET2's design is its hardware agnosticism which allows researchers and
+quantum software developers to take advantage of its state of the art
+compilation for many different quantum architectures.
+
+TKET2 circuits are represented using the HUGR IR defined in the
+[quantinuum-hugr] crate. The [`Circuit`] trait provides a high level interface
+for working with HUGRs representing quantum circuits, and defines a HUGR
```

Comment:
Phrasing/grammar: this `and` suggests that the subject on the left, i.e. `The [```Circuit```] trait`, defines the HUGR extension ;-). How about:
```
...[quantinuum-hugr] crate. TKET2 augments Hugr with
* The [`Circuit`] trait, providing a high-level interface for working with HUGRs representing quantum circuits
* a HUGR extension with quantum operations
* A composable pass system for optimising circuits
* A number of built-in passes for common optimisations
```